src/lib/database.js

```diff
@@ -0,0 +1,13 @@
+var Sequelize = require('sequelize');
+var sql;
+var sqlInited;
+
+module.exports = {
```

Comment:
that would mean no use of initSql arguments and instead using environment variables for dbUrl and log. Will do that on next release.